packages/coin-support-utils/tests/02.createAccount/02.generateDerivationPathPerScheme/__fixtures__/types.ts

```diff
@@ -0,0 +1,17 @@
+import { IGenerateDerivationPathsPerSchemeParams } from '../../../../src/createAccount';
```

Comment:
Always import from `src`

```suggestion
import { IGenerateDerivationPathsPerSchemeParams } from '../../../../src';
```